server/src/main/java/server/payload/request/SignupRequest.java

```diff
@@ -6,20 +6,33 @@
 public class SignupRequest {
     @NotBlank
     private String username;
+
+    @NotBlank
+    private String firstName;
```

Comment:
Ez és a lastName sem kerül ide, legalábbis a regisztrációs felület forráskódjában nem látom, hogy lennének. Szerintem ez csak későbbiekben a profil beállításainál állítható mint a profilkép.